TWLight/view_mixins.py

```diff
@@ -122,6 +126,29 @@ def dispatch(self, request, *args, **kwargs):
 
 
 
+class StaffOnly(object):
+    """
+    Restricts visibility to:
+    * Staff; or
+    * Superusers.
+    """
+
+
+    def test_func_staff_only(self, user):
+        return (user.is_superuser or user.is_staff)
```

Comment:
I don't think you need to check for both? I think superuser passes the staff check. don't take word for that.

Reply:
Ah, interesting. Like the comment above, I approached this from our eventual aim of 'staff' and 'superuser' being somewhat more distinct than they are now.

Reply:
Where our code is using django.contrib.auth, a superuser is treated as a user with all permissions assigned:
https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#django.contrib.auth.models.User.is_superuser

Staff just means they are able to see the admin site:
https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#django.contrib.auth.models.User.is_staff